src/frontend/screens/WineManager/index.css

```diff
@@ -0,0 +1,55 @@
+.wineManager {
+  .tabsWrapper {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+    align-items: center;
+
+    .FormControl__button {
+      width: fit-content;
+      cursor: pointer;
+
+      &:hover {
+        color: var(--accent);
+      }
+    }
+
+    .MuiTabs-root {
+      .MuiTabs-scroller {
+        .MuiTabs-indicator {
+          background-color: var(--accent);
+        }
+
+        .MuiTabs-flexContainer {
+          .MuiTab-root {
+            color: var(--text-default);
+          }
+
+          .Mui-selected {
+            color: var(--accent);
+          }
+        }
+      }
+    }
+  }
+
+  .wineList {
+    margin-top: 48px;
+
+    .gameListHeader {
+      grid-template-columns: 3fr 2fr 1fr 10%;
+      margin: 0 var(--space-xl-fixed);
+      & span:last-child {
+        justify-self: center;
+      }
+    }
+  }
+
+  .infoBox {
+    width: 80%;
+    margin: 0 auto;
+    & svg {
+      margin-inline-end: 0.5rem;
+    }
+  }
```

Comment:
this `.infoBox` is the only new thing in this file, the rest is just updated indentation (I added a `.wineManager` wrapper selector)